src/components/quotes.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react';
+
+export default function Quote() {
+  const [data, setData] = useState([]);
+  const [isPending, setIsPending] = useState(false);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsPending(true);
+      try {
+        const result = await fetch('https://api.api-ninjas.com/v1/quotes', {
+          method: 'GET',
+          headers: {
+            'X-Api-Key': 'EFksrx0o7WJjY2k4FvMs6A==Jx80OS03eWuEE1eC',
+            'Content-Type': 'application/json',
+          },
+        });
+        const json = await result.json();
+        setData(json);
+      } catch (error) {
+        setError(true);
+      }
+      setIsPending(false);
+    };
+    fetchData();
+  }, [setData]);
+
+  if (error) return <div>Unable to fetch quotes!</div>;
+  if (isPending) return <div>Loading...</div>;
+
+  return (
+    <div className="quotes">
+      <p>{data[0].quote}</p>
+      <span>{data[0].author}</span>
+    </div>
+  );
```

Comment:
- [ ] The app has an error, and it is preventing it from displaying the app content. Kindly take a look at the image below and make changes.

![qoutes](https://user-images.githubusercontent.com/53124548/229271532-eb0fb6af-1526-4bdd-abc5-807e943b067e.png)